extras/userdoc/md/documentation/quickref.md

```diff
@@ -174,7 +174,7 @@ Create a nested multi-dimensional network structure in the current subnet.
 
 ### Neuron Models
 
-[iaf\_neuron](../helpindex/cmds/iaf_neuron),
+[iaf\_neuron](../helpindex/cmds/iaf_psc_alpha),
```

Comment:
Yes, definitely. These were not matched by the regular expression I used for checking. Done in 3b9fb12f.